### DISCUSSION TITLE
Next.js + Tailwind CSS Starter Templates

### DISCUSSION THREAD

**Discussion by snelsi** [radix-ui/themes#353] [Show and tell]
Hello everyone! 👋

I'm excited to share two Next.js templates that I've been working on. 
They are designed to help you kickstart your projects with a solid foundation and a beautiful UI.

1. **Next.js + Radix Themes Template**: This template integrates Next.js with Radix Themes, providing a solid starting point for your projects. It's perfect for those who want to build a unique, custom UI without starting from scratch.

    Check it out here: https://github.com/snelsi/nextjs-radix-themes-template

2. **Next.js + Radix Themes + Tailwind CSS Template**: This template goes a step further by integrating Tailwind CSS — a utility-first CSS framework that is highly customizable and pairs well with Radix Themes. If you're a fan of Tailwind and want to leverage the power of Radix Themes, this template is for you.

    Check it out here: https://github.com/snelsi/nextjs-radix-themes-tailwind-template

Both templates are open-source and I welcome any feedback or contributions. I hope you find them useful for your projects!

Let me know what you think, and don't hesitate to ask if you have any questions or suggestions.

Happy coding! 🚀

**Comment by irfandyj** [comment]
Interesting, I've actually been looking for things like this. However, I were looking for something that has accessibility in mind. Some boilerplate for Radix and NextJS that allow me to copy-paste things up.

**Reply by snelsi** [reply]
I'm actually currently working on a UI kit to easily copy-paste premade sections and components into your app. But it's in a very early development stage.

Regarding the accessibility, what would you like to see in a template?
